.github/workflows/deploy.yaml

```diff
@@ -29,9 +31,16 @@ jobs:
         with:
           commit_message: "Bump app version [no ci]"
 
-      - name: Propagate master version bump to develop
+      - name: Propagate master version bump to develop if master version is ahead
         if: github.ref == 'refs/heads/master'
-        run: git push --force-with-lease origin HEAD:develop
+        run: |
+          git fetch origin develop
+          if [[ $(git diff --quiet HEAD~1 origin/develop) ]]; then
+            echo "Master version does not match develop version"
```

Comment:
Maybe if the `master` does not match the `develop` branch, we should create automatically create a PR to accommodate such fixes to the `develop` branch

Reply:
In this way we can adopt the fix workflow that we discussed in the past Thursday meeting: 

1. Make fix on the master branch
2. Make a PR into the master branch
3. Upon merging, the action will automatically create a PR to merge this changes into develop
4. If necessary, do some fixes to work in the develop branch
5. Upon approval, merge into develop

I think this is the most streamlined and error-free approach into getting fixes done in `master` but also in `develop`. 
Maybe this should be a seperate action workflow, and maybe done on a seperate issue, that listens to merged pull request events into `master, wdyt?

Reply:
With that said, maybe you should change the base of this PR to `master` and not to `develop` :sweat_smile: